src/rt/sections_android.d

```diff
@@ -155,14 +158,14 @@ ref void[] getTLSBlock() nothrow @nogc
 ref void[] getTLSBlockAlloc() nothrow @nogc
 {
     auto pary = &getTLSBlock();
-    if (!pary.length)
+    /*if (!pary.length)
     {
         auto pbeg = cast(void*)&_tlsstart;
         auto pend = cast(void*)&_tlsend;
         auto p = .malloc(pend - pbeg);
         memcpy(p, pbeg, pend - pbeg);
         *pary = p[0 .. pend - pbeg];
-    }
+    }*/
```

Comment:
Just remove it if it is no longer used.

Reply:
This and all subsequent comments are because this pull is a Work-In-Progess, as noted in OP. I'll clean it up once the functionality is set.